modules/client-configuration/pages/snippets/add_channels_scc.adoc

```diff
@@ -11,5 +11,6 @@ spacewalk-common-channels \
 . Synchronize the channels:
 +
 ----
-mgr-sync refresh --refresh-channels
+spacewalk-repo-sync
```

Comment:
If this SCC, why using spacewalk-repo-sync?

Reply:
Because for OSs that use spacewalk-channel-sync to add channels, they use spacewalk-repo-sync to synchronize:

![image](https://user-images.githubusercontent.com/3914967/98317755-d177c480-2028-11eb-859a-d3c0b7e972ec.png)